### ISSUE TITLE
I have a strong feeling that repo is dead

### ISSUE THREAD

**Issue by retardgerman** [nadimkobeissi/mkbsd#77]
I don't think any pull requests or issues are being processed here. It was still a good time. ✌️

**Comment by MalcolmReed-ent** [comment]
yeah, i mean what else is there to do, various scripts have been made and available.

**Comment by Kaushty** [comment]
@retardgerman @MalcolmReed-ent Is it still working ?
I get this error when trying to run it.
```Error: ⛔ Failed to fetch JSON file: Forbidden```

**Comment by retardgerman** [comment]
Yes, it looks like it's been fixed. Too bad, it was nice

**Comment by retardgerman** [comment]
> @retardgerman @MalcolmReed-ent Is it still working ? I get this error when trying to run it. `Error: ⛔ Failed to fetch JSON file: Forbidden`

Well, I got a workaround. checkout:  [my fork of the repo](https://github.com/retardgerman/mkbsd)

**Comment by PilotHaribo** [comment]
> Yes, it looks like it's been fixed. Too bad, it was nice

unfortunate :(
do you still have the wallpapers? mind hitting me up?

**Comment by retardgerman** [comment]
> > Yes, it looks like it's been fixed. Too bad, it was nice
> 
> 
> 
> unfortunate :(
> 
> do you still have the wallpapers? mind hitting me up?

My [Fork](https://github.com/retardgerman/mkbsd) is still working because my script uses a local JSON that I created while it was still possible. In there are the links to the Panels CDN with the images. But I can hit you up to if you like to.

**Comment by GlineYT** [comment]
https://www.mediafire.com/file/8ddaiou56u56i7y/MKBSD.tar/file
Here is a archive of all the images

**Comment by nor0x** [comment]
my web app https://github.com/nor0x/OpenPanels is also still displaying the full res images. also got a local copy of ~12GB of images
![image](https://github.com/user-attachments/assets/f5f04969-68e7-4114-9944-2eb377f5af64)

**Comment by GlineYT** [comment]
12GB? When did they add moire? When i ran the script only about 300 or so images downloaded.
I might back them up, incase this gets taken down, but honestly i do value my storage space, and 12GB for ai generated slop/poorly made "abstract" "art" just ain't it, chief. Like, that ain't a good deal.
